HyperClick.sublime-settings

```diff
@@ -35,7 +35,7 @@
     // - aliases map something short in the import statement (e.g. "@") to a path in your project
     // - HyperClick will also search in lookup_paths, which is particularly useful in project settings
     "scopes": {
-        "source.ts, source.js, source.tsx, source.jsx": {
+        "source.js | source.jsx | source.ts | source.tsx": {
```

Comment:
I think `|` with spaces around has better readability due to extra spaces. But I can revert this if it's not preferred for whatever reason.

Reply:
I'm open to this change, but for consistency all other selectors (`scope_is_js` / `scope_is_sass` etc) should be changed

Reply:
resolved in https://github.com/aziz/SublimeHyperClick/pull/78/commits/8e91ba5bd034f691f1fe34f8352149bb7d10a2d0